### DISCUSSION TITLE
wcc386 bug, v 2.0 beta, initializing OVERLAPPED struct

### DISCUSSION THREAD

**Discussion by HnkGitHub** [open-watcom/open-watcom-v2#1141] [General]
Git Version 9/23/23, 64-bit
Platform Windows 10
Building a Win32s/Win32 console exe.

I'm getting an error when compiling an object. The code tries to initialize an overlapped structure.
The same file builds fine on Watcom 1.9.
I need to initialize the structure to set the proper beginning offset.
```

#include <windows.h>
int _tmain()
{
    OVERLAPPED ovl ;
    ovl.Offset = 0 ;
......
`
```
Please, see the attached files.

Thank you,

Hank
[ccerror.zip](https://github.com/open-watcom/open-watcom-v2/files/12710920/ccerror.zip)

**Comment by jmalak** [comment]
Sorry, what exact problem is?
I cannot get any problem, on my box it compiles without problem.
Do you have setup properly following environment variables to point to right OW version?
WATCOM
PATH
INCLUDE

**Reply by HnkGitHub** [reply]
Well, it works now on the Sept. 29, 2023 build. Somebody must have fixed  it.

Thanks

**Reply by jmalak** [reply]
Sorry, it is not possible, build system do tests that such stupid error simply can not happen.
